static/app/components/charts/components/tooltip.tsx

```diff
@@ -224,8 +225,6 @@ function getFormatter({
       `<div class="tooltip-date">${date}</div>`,
       getTooltipArrow(),
     ].join('');
-
-    return `<div class="tooltip-container">${tooltipContent}</div>`;
```

Comment:
ECharts V5 now wraps all of its tooltips inside containers, so we no longer need to manually do that.